notebooks/community/sdk/sdk_pytorch_torchrun_custom_container_training_imagenet.ipynb

```diff
@@ -0,0 +1,2732 @@
+{
```

Comment:
<p>Title: sentence case and branding:</p><h1>PyTorch multi-node distributed training with Torchrun on Vertex AI Training using a custom container </h1><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1323/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>